cmd/influx_inspect/verify/tombstone/verify.go

```diff
@@ -1,142 +0,0 @@
-// Package tombstone verifies integrity of tombstones.
-package tombstone
-
-import (
-	"errors"
-	"flag"
-	"fmt"
-	"io"
-	"os"
-	"path/filepath"
-	"time"
-
-	"github.com/influxdata/influxdb/v2/tsdb/engine/tsm1"
-)
-
-// Command represents the program execution for "influx_inspect verify-tombstone".
-type Command struct {
-	Stderr io.Writer
-	Stdout io.Writer
-}
-
-// NewCommand returns a new instance of Command.
-func NewCommand() *Command {
```

Comment:
This was all dead code? That's unfortunate.

Reply:
I was able to repurpose much of it, but yeah it is dead code on `master` right now.